api/src/main/java/org/pmiops/workbench/access/AccessModuleMapper.java

```diff
@@ -0,0 +1,32 @@
+package org.pmiops.workbench.access;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingConstants;
+import org.mapstruct.ValueMapping;
+import org.pmiops.workbench.actionaudit.targetproperties.BypassTimeTargetProperty;
+import org.pmiops.workbench.db.model.DbAccessModule.DbAccessModuleName;
+import org.pmiops.workbench.model.AccessModule;
+import org.pmiops.workbench.model.AccessModuleName;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+@Mapper(config = MapStructConfig.class)
+public interface AccessModuleMapper {
+  @Deprecated // use the AccessModuleName mappers
```

Comment:
yes I'm introducing something that's immediately deprecated.  This is to support the transition away from the old `AccessModule`